docs/Demos.md

```diff
@@ -29,6 +219,30 @@ The source shows how to implement the following features:
 - [Custom page with static list](https://github.com/marmelab/react-admin/blob/master/examples/demo/src/routes.tsx)
 - [Edit view in a sidebar](https://github.com/marmelab/react-admin/blob/master/examples/demo/src/reviews/ReviewList.tsx)
 
+## E-commerce enterprise version
+
+Based on the E-commerce demo and upgraded with some ReactAdmin-Enterprise modules.
+
+<video src="https://github.com/marmelab/react-admin/assets/131013150/93ec41f5-81bf-45e1-940b-d5aeccfe628f" controls="controls" style="max-width: 100%"></video>
```

Comment:
Why not use the same tags (and style) as other videos in the docs?

In particular, it seems the video player is a but cropped to the right.

![image](https://github.com/marmelab/react-admin/assets/14542336/ab974bf7-bd54-4996-85af-b5bab552bbfe)

Also, can you not include the browser frame in the video?

Lastly, apart from a few exceptions, it seems to me you are not focusing enough on the Enterprise features included in this demo, and you are instead demonstrating mainly the same features as the OSS version of this demo. We can have a look together of the Enterprise features we should highlight if you need.